### ISSUE TITLE
Tooltip misalignment w/ Horizontal Scrolling

### ISSUE THREAD

**Issue by benkennedysnc** [twbs/bootstrap#15590]
MacBook Pro 2.3 ghz Core i7
Mac OSX 10.9.5
Google Chrome (version 42.0.2278.0 canary (64-bit))
Safari (version 7.1 (9537.85.10.17.1))
Mozilla Firefox (version 34.0.5)

When creating wide data tables with many td elements, the tooltip does not align with corresponding far right td elements when the frame is scrolled horizontally.  The td elements that reproduce the issue must be made visible by scrolling horizontally.

http://jsfiddle.net/benkennedy/Ld0ptsov/1/

1.  Load the JSFiddle page
2.  Shrink the fourth pane (bottom right pane) to the point where you only see half of the table.
3.  Hover on the first td element, and notice the tooltip is working as expected.
4.  Scroll horizontally to the far right, and hover over the last td element.  Notice the tooltip is not in the expected location.

This issue looks like it may be located in the Tooltip.prototype.getPosition method, more specifically - width: $(window).width().

**Comment by twbs-lmvtfy** [comment]
Hi @benkennedysnc!

You appear to have posted a live example (http://fiddle.jshell.net/benkennedy/5uprtsrc/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 12, column 3: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.1 ; saw what appears to be usage of Bootstrap 3.2.0
* line 13, column 3: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.1 ; saw what appears to be usage of Bootstrap 3.2.0

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by benkennedysnc** [comment]
http://jsfiddle.net/benkennedy/Ld0ptsov/1/

**Comment by cvrebert** [comment]
Confirmed on OS X in Safari 8.0.2, Chrome, and Firefox.

**Comment by ericmartel** [comment]
This might be fixed by this PR https://github.com/twbs/bootstrap/pull/15494

I'm at work, I'll check it when I get home tonight

**Comment by ericmartel** [comment]
I confirm that PR #15494 does not fix the issue.  If I have time today I will investigate the issue further.

**Comment by xak2000** [comment]
Confirmed on Win7 in last versions of Chrome and Firefox.

**Comment by mdo** [comment]
Punting as a won't fix—v4 includes a new positioning library for our affix, tooltips, and popovers.

**Comment by t0il3ts0ap** [comment]
Is there any quick fix for this issue ?

**Comment by cvrebert** [comment]
#15686, perhaps.
